term.go

```diff
@@ -216,6 +218,14 @@ func (t *Terminal) Text() string {
 	return t.content.Text()
 }
 
+// ExitCode returns the exit code from the terminal's shell
```

Comment:
Looking good thanks, but it omits the info about -1 before the terminal command has exited, can you include it in the docs?

Reply:
done (in README) - hope it's not too verbose...
Thanks!

Reply:
I was meaning here in the API documentation. Folk basically never use README for API documentation - this section linked is what the IDE will look up during implementation.

Reply:
makes sense - sorry I got it wrong - fixed